### PR TITLE
fix README

### PR DESCRIPTION
**Pull request by boguslavsky**
1. Fix typos in the table of Unsigned Integers
2. Fix Transaction example

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -402,9 +402,9 @@ For unsafe numbers use `string` only.*
 | Name | Type | Size (bytes) | Description |
 |---|---|---|---|
 | Uint8 | `number` | `1` | Signed integer in a range from `0` to `255` |
-| Int16 | `number` | `2` | Signed integer in a range from `0` to `65535` |
-| Int32 | `number` | `4` | Signed integer in a range from `0` to `4294967295` |
-| Int64 | `number` or `string`* | `8` | Signed integer in a range from `0` to `18446744073709551615` |
+| Uint16 | `number` | `2` | Signed integer in a range from `0` to `65535` |
```
Comment (slowli) [inline README.md]: It's "unsigned integer", isn't it?

File: README.md [hunk 2]
```diff
@@ -170,17 +170,17 @@ Here is an example of 'SendFunds' transaction:
 
 ```javascript
 var SendFunds = Exonum.newMessage({
```
Comment (slowli) [inline README.md]: Could you please add a unit test for this snippet (and possibly other examples from the README) to ensure that it actually works?

Comment (boguslavsky) [reply README.md]: Done: https://github.com/exonum/exonum-client/pull/60/files#diff-7130f0fbba2575608a871f35ee60d08a

File: test/readme.js [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+            name: 'Tom',
+            age: 34
+        };
+        var secretKey = '978e3321bd6331d56e5f4c2bdb95bf471e95a77a6839e68d4241e7b0932ebe2bfa7f9ee43aff70c879f80fa7fd15955c18b98c72310b09e7818310325050cf7a';
+        var signature = Exonum.sign(secretKey, data, User);
+
+        expect(signature).to.equal('c1db9a5f01ebdff27e02652a9aae5c9a4ac88787587dabceb9f471ae0b8e051b9632dfd26922f6abf24ff2d3275028fe286703d25ee7fe6b1711e89af4a7d307');
+    });
+
+    it('should verify custom type signature', function() {
```
Comment (slowli) [inline test/readme.js]: You can probably combine this test case with the previous one, either as a single test case, or as IIFE:

```javascript
(function() {
  var User = ...;
  var data = ...;
  var secretKey = ...;
  var publicKey = ...;

  it('should sign custom type', function() {
    expect(Exonum.sign(secretKey, data, User)).to.equal(...);
  });

  it('should verify custom type signature', function() {
    var signature = ...;
    expect(Exonum.verifySignature(signature, publicKey, data, User)).to.be.true;
  });
})();
```

File: test/readme.js [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+            }
+        });
+        var data = {
+            name: 'Tom',
+            age: 34
+        };
+        var publicKey = 'fa7f9ee43aff70c879f80fa7fd15955c18b98c72310b09e7818310325050cf7a';
+        var signature = 'c1db9a5f01ebdff27e02652a9aae5c9a4ac88787587dabceb9f471ae0b8e051b9632dfd26922f6abf24ff2d3275028fe286703d25ee7fe6b1711e89af4a7d307';
+
+        expect(Exonum.verifySignature(signature, publicKey, data, User)).to.equal(true);
```
Comment (slowli) [inline test/readme.js]: Nitpick: you may use `.to.be.true` (or `to.be.true()` if you use [`dirty-chai`](https://www.npmjs.com/package/dirty-chai) to get rid of possible linter warnings).

File: test/readme.js [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+            to: 'f5864ab6a5a2190666b47c676bcf15a1f2f07703c5bcafb5749aa735ce8b7c36',
+            amount: 50
+        };
+        var secretKey = '978e3321bd6331d56e5f4c2bdb95bf471e95a77a6839e68d4241e7b0932ebe2bfa7f9ee43aff70c879f80fa7fd15955c18b98c72310b09e7818310325050cf7a';
+        var signature = SendFunds.sign(secretKey, data);
+
+        expect(signature).to.equal('8ff692b47d17f7738ec2d19f5296d3810909a62fe1480eb752b531f8e17056f6578985090db822612f88c3bcad4f0539401836e5ad58913c489784ff3e415a0b');
+    });
+
+    it('should verify transaction signature', function() {
```
Comment (slowli) [inline test/readme.js]: Could be combined with the previous test case (see the previous comment).

File: test/readme.js [hunk 6]
```diff
@@ -0,0 +1,72 @@
+/* eslint-env node, mocha */
+
+var expect = require('chai').expect;
+var Exonum = require('..');
+
+describe('Examples from README.md', function() {
```
Comment (slowli) [inline test/readme.js]: Nitpick: some linters mark as warning fragments that are delimited by empty lines from both sides. Propose to format as follows:

```javascript
describe('Examples from README.md', function() {
    describe('Custom type section', function() {
        var User = Exonum.newType({
        // other lines
    });

    describe('Transaction section', function() {
        // ...
    });
});
```

Comment (boguslavsky) [reply test/readme.js]: done, thanks

**Pull request merged**